.github/workflows/build-windows.yml

```diff
@@ -16,19 +20,15 @@ jobs:
       BOOST_EXE: boost_1_72_0-msvc-14.2
 
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Can you please explain the rationale behind changing this?

Reply:
Yes. Because I don't want to fail Release or Debug if the other one is fail. Because they can fail from other reasons. So it better let them to fail by their own.